src/Database/DataFeed.php

```diff
@@ -211,10 +216,17 @@ protected function processCollection()
             /*
              * Flush the select, add ID and tag
              */
-            $cleanQuery = $cleanQuery->select(Db::raw($keyName." as id"));
-            $cleanQuery = $cleanQuery->addSelect(Db::raw("(SELECT '".$tag."') as ".$this->tagVar));
-            $cleanQuery = $cleanQuery->addSelect(Db::raw("(SELECT ".$sorting.") as ".$this->sortVar));
 
+            $conditionalTagSelect = (Db::connection()->getDriverName() === 'pgsql') ?
+                "CAST('%s' as text) as %s" :
+                "%s as %s";
```

Comment:
@dimensi I think this line without quotes throws an error. Should be the same as for pqsql, but without casting: `"'%s' as %s"`.

Reply:
fix it